package.json

```diff
@@ -94,6 +94,7 @@
   "dependencies": {
     "@backstage/errors": "workspace:^",
     "@manypkg/get-packages": "^1.1.3",
+    "@scaffolder/cli": "0.1.0-next.0",
```

Comment:
Just a pedantic detail, but I think it makes sense to keep this one in devDependencies, not that it really matters but that's where we have the other CLIs